src/domains/citizen-auth-app/08_session_manager.tf

```diff
@@ -352,6 +352,112 @@ module "session_manager_weu_staging" {
   tags = var.tags
 }
 
+module "session_manager_weu_bis" {
+  source = "github.com/pagopa/terraform-azurerm-v3//app_service?ref=v8.28.1"
+
+  # App service plan
+  plan_type              = "internal"
+  plan_name              = format("%s-session-manager-asp-04", local.common_project)
+  zone_balancing_enabled = true
+  sku_name               = var.session_manager_plan_sku_name
+
+  # App service
+  name                = "${local.app_name_weu}-04"
+  resource_group_name = data.azurerm_resource_group.session_manager_rg_weu.name
+  location            = var.location
+
+  always_on    = true
+  node_version = "20-lts"
+  # NOTE:
+  # 1. index.js file is generated from the deploy pipeline
+  # 2. the linux container for app services already has pm2 installed
+  #    (refer to https://learn.microsoft.com/en-us/azure/app-service/configure-language-nodejs?pivots=platform-linux#run-with-pm2)
+  app_command_line             = "pm2 start index.js -i max --no-daemon"
+  health_check_path            = "/healthcheck"
+  health_check_maxpingfailures = 2
+
+  auto_heal_enabled = true
+  auto_heal_settings = {
+    startup_time           = "00:05:00"
+    slow_requests_count    = 50
+    slow_requests_interval = "00:01:00"
+    slow_requests_time     = "00:00:10"
+  }
+
+  app_settings = merge(
+    local.app_settings_common,
+    {
+      APPINSIGHTS_CLOUD_ROLE_NAME = "${local.app_name_weu}-04"
+    }
+  )
+  sticky_settings = concat(["APPINSIGHTS_CLOUD_ROLE_NAME"])
+
+
+  allowed_subnets = [
+    data.azurerm_subnet.appgateway_snet.id,
+    data.azurerm_subnet.apim_itn_snet.id,
+    // TODO: add proxy subnet
```

Comment:
can this be removed? The new session-manager would be reachable via the private endpoint created above, do you agree @BurnedMarshal?